packages/protoc/package.json

```diff
@@ -19,7 +19,10 @@
   "bin": {
     "protoc": "./protoc.js"
   },
+  "peerDependencies": {
+    "proxy-agent": "^5.0.0"
+  },
```

Comment:
You're right, I was thinking of this package being installed in a broader host context but since this is a hard requirement of this installation script making it a peer dependency does not make much sense.